### PR TITLE
fix keycloak import filename

### PR DESCRIPTION
**Pull request by bhalsey**

### CODE REVIEW

File: site/docs/v1/tech/migration-guide/keycloak.adoc [hunk 1]
```diff
@@ -242,9 +242,9 @@ When you run the script, you'll see output like:
 
 [source,shell,title=Import script output]
 ----
-$ ruby ./import.rb -f http://localhost:9011 -k '...' -u users.json
+$ ruby ./import.rb -f http://localhost:9011 -k '...' -u out.csv
```
Comment (mooreds) [inline site/docs/v1/tech/migration-guide/keycloak.adoc:245]:
```suggestion
$ ruby ./import.rb -f http://localhost:9011 -k '...'
```

Comment (mooreds) [reply site/docs/v1/tech/migration-guide/keycloak.adoc:245]: The default for `-u` is `out.csv`

**Review by mooreds — APPROVED** [review]
One small change suggested.

File: site/docs/v1/tech/migration-guide/keycloak.adoc (hunk 1 above)
Comment (bhalsey) [reply site/docs/v1/tech/migration-guide/keycloak.adoc:245]: I like keeping it on the command line to help show how the steps connect together.  Also helpful if customers name their file something else.

**Pull request merged**